src/services/dashboards/dashboard-customize/modules/dashboard-manage-variable-overlay/DashboardManageVariableOverlay.vue

```diff
@@ -114,21 +117,41 @@ const handleChangeAddContent = () => {
     state.contentType = 'ADD';
 };
 const handleOpenDeleteModal = (propertyName: string) => {
-    if (state.contentType === 'LIST') state.selected = propertyName;
+    if (state.contentType === 'LIST') state.selectedVariable = propertyName;
     deleteModalState.visible = true;
 };
 const handleDeleteVariable = () => {
-    console.log(state.selected, 'Delete!!!!');
+    const properties = cloneDeep(props.variables) as DashboardVariablesSchema['properties'];
+    delete properties[state.selectedVariable];
+    const changedOrder = props.order.filter((d) => d !== state.selectedVariable);
+    emit('change', properties, changedOrder);
+
+    if (state.contentType === 'EDIT') state.contentType = 'LIST';
+    state.selectedVariable = '';
+    deleteModalState.visible = false;
 };
 const handleChangeVariableUse = (name: string, value: boolean) => {
     const properties = cloneDeep(props.variables) as DashboardVariablesSchema['properties'];
     properties[name].use = value;
     emit('change', properties);
 };
 const handleChangeEditContent = (propertyName: string) => {
-    state.selected = propertyName;
+    state.selectedVariable = propertyName;
     state.contentType = 'EDIT';
 };
+const handleSaveVariable = (variable: DashboardVariableSchemaProperty) => {
+    const properties = cloneDeep(props.variables) as DashboardVariablesSchema['properties'];
+    if (contentType.value === 'ADD') {
```

Comment:
nitpick: This is more friendly :)

```suggestion
    if (state.contentType=== 'ADD') {
```